README.md

```diff
@@ -90,7 +89,15 @@ Or, with a poe task:
 
 It is recommended to run Kyma Companion with Poetry because it activates and uses its virtual environment if not activated yet.
 
-> [!NOTE] You cannot run it with Python directly.
+As an alternative, you can run the application with the following command directly via Python, using [`uvicorn`](https://www.uvicorn.org/) instead of `FastAPI`:
```

Comment:
```suggestion
Alternatively, you can run the application directly using Python and [`uvicorn`](https://www.uvicorn.org/) instead of `FastAPI`. To do this, run the following command:
```